src/hashtree.erl

```
@@ -631,7 +631,7 @@ get_disk_bucket(Level, Bucket, #state{id=Id, ref=Ref}) ->
 set_disk_bucket(Level, Bucket, Val, State=#state{id=Id, ref=Ref}) ->
     HKey = encode_bucket(Id, Level, Bucket),
     Bin = term_to_binary(Val),
-    eleveldb:put(Ref, HKey, Bin, []),
+    ok = eleveldb:put(Ref, HKey, Bin, []),
```

@jtuple are we ok w/ semantic change here that we may crash if failing to write to leveldb instead of silently ignoring it? I _think_ its ok, but wanted to double check. we could also just change `ok =` to `_ =`...